olive/systems/isolated_ort/isolated_ort_system.py

```diff
@@ -128,7 +129,7 @@ def _run_inference(
         output_dir: Union[str, Path],
     ):
         command = [
-            "python",
+            sys.executable,
```

Comment:
this cannot be sys.executable since it points to the python executable of the engine. We need the python executable in the environment.

Reply:
if we want a full path, we could add an attribute in the init like `self.executable = shutil.which(...)` and use it.

Reply:
let's see whether the CI fail or not. If not fail, then it is a bug.

Reply:
The ci will probably pass since the test environment has all of the packages installed. We only test for cpu package I think.